### ISSUE TITLE
Debian build tests fail with missing vendor/github.com/xeipuuv/gojsonschema/json_schema_test_suite/type/schema_0.json

### ISSUE THREAD

**Issue by javabrett** [github/git-lfs#1290]
Recent changes to vendor deps might interact with the Debian build - at least I have not seen this one before.  Could also be my environment - someone can deb build master and confirm.

```
$ dpkg-buildpackage -us -uc
...
=== RUN   TestJsonSchemaTestSuite
Test (0) | integer type matches integers :: an integer is an integer
--- FAIL: TestJsonSchemaTestSuite (0.00s)
	schema_test.go:379: Error (open <redacted>/go/src/github.com/github/git-lfs/obj-x86_64-linux-gnu/src/github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonschema/json_schema_test_suite/type/schema_0.json: no such file or directory)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x8 pc=0x4b69bb]

goroutine 6 [running]:
panic(0x8073e0, 0xc820010170)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
testing.tRunner.func1(0xc8200201b0)
	/usr/local/go/src/testing/testing.go:467 +0x192
panic(0x8073e0, 0xc820010170)
	/usr/local/go/src/runtime/panic.go:426 +0x4e9
github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonschema.TestJsonSchemaTestSuite(0xc8200201b0)
	<redacted>/go/src/github.com/github/git-lfs/obj-x86_64-linux-gnu/src/github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonschema/schema_test.go:381 +0xe57b
testing.tRunner(0xc8200201b0, 0xabf958)
	/usr/local/go/src/testing/testing.go:473 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:582 +0x892
FAIL	github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonschema	0.010s
dh_auto_test: go test -v github.com/github/git-lfs github.com/github/git-lfs/api github.com/github/git-lfs/api/schema github.com/github/git-lfs/auth github.com/github/git-lfs/commands github.com/github/git-lfs/config github.com/github/git-lfs/docs/man github.com/github/git-lfs/errutil github.com/github/git-lfs/git github.com/github/git-lfs/httputil github.com/github/git-lfs/lfs github.com/github/git-lfs/localstorage github.com/github/git-lfs/progress github.com/github/git-lfs/script github.com/github/git-lfs/subprocess github.com/github/git-lfs/tools github.com/github/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm github.com/github/git-lfs/vendor/github.com/ThomsonReutersEikon/go-ntlm/ntlm/md4 github.com/github/git-lfs/vendor/github.com/bgentry/go-netrc/netrc github.com/github/git-lfs/vendor/github.com/cheggaaa/pb github.com/github/git-lfs/vendor/github.com/cheggaaa/pb/example github.com/github/git-lfs/vendor/github.com/cheggaaa/pb/example/copy github.com/github/git-lfs/vendor/github.com/inconshreveable/mousetrap github.com/github/git-lfs/vendor/github.com/kr/pretty github.com/github/git-lfs/vendor/github.com/kr/pty github.com/github/git-lfs/vendor/github.com/kr/text github.com/github/git-lfs/vendor/github.com/kr/text/colwriter github.com/github/git-lfs/vendor/github.com/kr/text/mc github.com/github/git-lfs/vendor/github.com/olekukonko/ts github.com/github/git-lfs/vendor/github.com/rubyist/tracerx github.com/github/git-lfs/vendor/github.com/spf13/cobra github.com/github/git-lfs/vendor/github.com/spf13/pflag github.com/github/git-lfs/vendor/github.com/technoweenie/go-contentaddressable github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonpointer github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonreference github.com/github/git-lfs/vendor/github.com/xeipuuv/gojsonschema returned exit code 1
make[1]: *** [override_dh_auto_test] Error 1
make[1]: Leaving directory `<redacted>/go/src/github.com/github/git-lfs'
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```

`vendor/github.com/xeipuuv/gojsonschema/json_schema_test_suite/type/schema_0.json` is indeed missing after the copy to the deb build dir, and what gets copied from the vendor directory by `dh_golang` is just the `*.go` files.  This is evident in the `dh_golang` copy logic, and you can set `DH_GOLANG_INSTALL_ALL=1` as in https://github.com/javabrett/git-lfs/commit/0367ee1ee219a71d16a959f8f385124bd1b0dcbb and it will copy the JSON file - you will get different errors then:

```
--- FAIL: TestFile (0.00s)
	file_test.go:171: <redacted>/go/src/github.com/github/git-lfs/obj-x86_64-linux-gnu/src/github.com/github/git-lfs/vendor/github.com/technoweenie/go-contentaddressable/file_test.go:37
		Expected: <nil>
		Actual:   invalid argument
```

But before proceeding - is this just running tests within vendor code that needs to be excluded?

**Comment by javabrett** [comment]
I notice that a series of exclusions have been applied in `script/test` by 28541a5, 28541a5 and specifically 9164f83.  I guess that these now need to be worked into the Debian build somehow.  Probably via some judicious `rm`ing, since I don't know that `dh_golang` is going to offer this level of include/exclude sophistication?

**Comment by technoweenie** [comment]
Nice catch. Is this fixed after merging #1291?

**Comment by javabrett** [comment]
Yes this is fixed. :heavy_check_mark:
